src/components/Tabs/Tabs.tsx

```diff
@@ -42,17 +42,19 @@ export const Tabs = ({
       sx={containerStyles}
     >
       {data.map((el, index) => {
+        const keys = a11yProps(ariaLabel, index);
         const tab = (
           <Tab
-            key={`${el.label ?? 'tab-key'}-${index}`}
+            className={`${ariaLabel} ${value === index ? 'active' : 'inactive'}`}
+            key={`${keys.id}-${index}`}
             onClick={(event) => {
```

Comment:
This should throw a ts warning as not typed

Reply:
Hmm, I checked but do not get an error here. Should be fine?